### ISSUE TITLE
Unpublished articles on featured article list

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#5529]
We rely on sitemaps to show 'mutliple ways' as if every front of house page is also on the sitemap, then it can be reached via inpage links and via the sitemaps (at least two ways). 

Sitemaps have been rewritten. Testing with playwright to crawl a test site, shows one mismatch between sitemaps and the crawled front of house.  Accepted but not yet published articles are missing in the sitemap. Further testing shows these cannot be guaranteed to have more than one link to them from elsewhere.

---

Further investigation
This is about unpublished articles on the front of house that shouldn't be there.  They should not be appearing as options for the featured articles list.

**Comment by StephDriver** [comment]
<img width="546" height="742" alt="Image" src="https://github.com/user-attachments/assets/b832db97-0055-4d88-a601-a6ba221cee84" />

this was the source of the trouble. It allowed unpublished articles to appear on the featured articles list when they are excluded elsewhere on the front of house.

**Comment by mauromsl** [comment]
closed by #5532
